packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -115,12 +119,19 @@ export const ExtensionApiExplorer: FC = () => {
     if (sdk && !specs) loadSpecs().catch((err) => console.error(err))
   }, [specs, sdk])
 
+  const extensionEnvAdaptor = new ExtensionEnvAdaptor(getExtensionSDK())
+  const store = configureStore()
```

Comment:
Are we going to have issues with this when the route changes (same as we had with the standalone). Been thinking about that solution and perhaps useMemo is a better solution

Reply:
Good point. I'll find some time today for us to talk about this

Reply:
Moved store creation outside of the component to avoid this happening